.golangci.yml

```diff
@@ -0,0 +1,117 @@
+linters:
+  enable:
+    - gci
+    - godot
+    - gofmt
+    - misspell
+    - whitespace
+    - revive
+issues:
+  exclude-use-default: false
+linters-settings:
+  gci:
+    sections:
+      - standard
+      - default
+      - prefix(github.com/canonical/microcluster)
+  errcheck:
+    exclude-functions:
+      - (io.ReadCloser).Close
+      - (io.WriteCloser).Close
+      - (io.ReadWriteCloser).Close
+      - (*os.File).Close
+      - (*github.com/gorilla/websocket.Conn).Close
+      - (*github.com/mdlayher/vsock.Listener).Close
```

Comment:
Should we add some comments explaining why those are excluded?

Reply:
Ah yes these were commonly ignored errors in LXD. I'll double check these.

Reply:
I've removed all of these except for `(io.ReadCloser).Close` which is used for closing response bodies. We can discuss adding more again in future.